docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -198,6 +198,55 @@ It quietly succeeds. Now `\d children` shows that the foreign key constraint `ch
 
 Add the specified constraint to the table.
 
+#### [*alter_column_type*]
+
+Change the type of an existing column. If data on disk is required to change, a full table rewrite is needed.
+
+The new column type will be applied automatically to indexes and simple table constraints by casting the initially provided expression.
+
+If the optional `COLLATE` clause is not specified, the default collation for the new column type will be used.
+
+If the optional `USING` clause is not provided, the default conversion for the new column value will be the same as an assignment cast from the old type to the new type.
+
+A `USING` clause must be included when there is no implicit or assignment cast available from the old type to the new type.
```

Comment:
> implicit or assignment cast

Did you mean "implicit assignment cast"?  What is the difference between "implicit assignment cast" and "assignment cast" and "implicit cast"?

Reply:
Changed in https://github.com/yugabyte/yugabyte-db/pull/17470/commits/78f22e5f78b55b21a5563fd8596a46b11b0997d6